.github/workflows/OWNERS

```diff
@@ -0,0 +1,10 @@
+options:
+  no_parent_owners: true
```

Comment:
we don't need this here. The top-level approvers can approve.

Reply:
Removed